.travis.yml

```diff
@@ -20,3 +22,10 @@ after_success:
 notifications:
   irc:
   - "chat.freenode.net#wet-boew"
+
+cache:
+  bundler: true
+  directories:
+  - node_modules
+  - lib
+  - vendor/cache
```

Comment:
Does this folder exists anywhere? Also how does it know how to invalidate the cache?